modules/navigation/3d/nav_mesh_generator_3d.cpp

```diff
@@ -151,25 +152,34 @@ void NavMeshGenerator3D::bake_from_source_geometry_data(Ref<NavigationMesh> p_na
 		if (p_callback.is_valid()) {
 			generator_emit_callback(p_callback);
 		}
+		p_navigation_mesh->emit_changed();
 		return;
 	}
 
 	if (is_baking(p_navigation_mesh)) {
 		ERR_FAIL_MSG("NavigationMesh is already baking. Wait for current bake to finish.");
 	}
 	baking_navmesh_mutex.lock();
-	baking_navmeshes.insert(p_navigation_mesh);
+	NavMeshGeneratorTask3D *generator_task = memnew(NavMeshGeneratorTask3D);
```

Comment:
This seem small enough to be on stack ? Why do you need memnew ?

Reply:
(as you delete it just below)

Reply:
yes